src/CoreWCF.Primitives/src/CoreWCF/Channels/ChannelDemuxer.cs

```diff
@@ -48,21 +48,21 @@ public int MaxPendingSessions
             }
         }
 
-       internal IServiceDispatcher CreaterServiceDispatcher<TChannel>(IServiceDispatcher innerDispatcher, ChannelDemuxerFilter filter)
+       internal IServiceDispatcher CreaterServiceDispatcher<TChannel>(IServiceDispatcher innerDispatcher, ChannelDemuxerFilter filter, BindingContext context)
         {
```

Comment:
As I mentioned in the other comment, typo in method name here

Reply:
done